src/main.rs

```diff
@@ -28,9 +30,10 @@ fn print_version() {
 }
 
 fn print_usage(opts: Options) {
-    let usage = concat!("\n\tbuild -- build the cobalt project at the source dir",
+    let usage = concat!("\n\tbuild -- build the cobalt project at the source dir, -i --import will import the site to the gh-pages branch",
```

Comment:
@johannhof so you can run `cobalt build -b src -d build --import` and it will build then import. It would be useful for ci tools to do it all in one command.